# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webcodecs/VideoDecoder.h
# Commit: 8f3c10480da7
# Full Hash: 8f3c10480da797e1ece28c96bad288d98a02e2df
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2023-06-29 15:31:45
# Regressor Bug: 1831451
# File Overlap Count: 1
# Description:
#   Bug 1831451 - Implement VideoDecoder::IsConfigSupported r=padenot
#   
#   This patch implements the isConfigSupported method for WebCodecs'
#   VideoDecoder.
#   
# ==============================================================================

diff -r eb17a0f3efdf -r 8f3c10480da7 dom/media/webcodecs/VideoDecoder.h
--- a/dom/media/webcodecs/VideoDecoder.h	Thu Jun 29 04:15:00 2023 +0000
+++ b/dom/media/webcodecs/VideoDecoder.h	Thu Jun 29 04:15:01 2023 +0000
@@ -71,7 +71,7 @@
   void Close(ErrorResult& aRv);
 
   static already_AddRefed<Promise> IsConfigSupported(
-      const GlobalObject& global, const VideoDecoderConfig& config,
+      const GlobalObject& aGlobal, const VideoDecoderConfig& aConfig,
       ErrorResult& aRv);
 };
 